charts/datadog/templates/_container-agent.yaml

```
@@ -264,6 +264,26 @@
     - name: host-rpm-dir
       mountPath: /host/var/lib/rpm
       readOnly: true
+    {{- if ne .Values.datadog.osReleasePath "/etc/redhat-release" }}
```

I think we should us `eq` instead of `ne` here in order to mount the redhat-release volume if the user set it in the `.Values.datadog.osReleasePath`
WDYT?

```suggestion
    {{- if eq .Values.datadog.osReleasePath "/etc/redhat-release" }}
```

---

also instead of adding 4 if condition can we something more generic

something like 

```yaml
    {{- if ne .Values.datadog.osReleasePath "" }}
    - name: etc-os-release
      mountPath: /host/{{.Values.datadog.osReleasePath}}
      readOnly: true
    {{- end }}

```

---

After an offline discussion with @lebauce . `ne` is the expected behaviour since we don't want to mount twice the same path if never the user has set a path in `.Values.datadog.osReleasePath`

---

This is intended (and shamelessly copy pasted from somewhere else in the charts).
The idea is to avoid having the volumes to be defined twice in the case where datadog.osReleasePath is set to /etc/redhat-release.

---

Chips